tests/net/socket/poll/prj.conf

```diff
@@ -21,3 +21,6 @@ CONFIG_MAIN_STACK_SIZE=2048
 CONFIG_ZTEST=y
 
 CONFIG_QEMU_TICKLESS_WORKAROUND=y
+
```

Comment:
What exact problem you have without these options? This tests is supposed (and does) run fine without these options with standard Zephyr qemu networking setup. And relying on such setup makes it easier to hack on this test (and adds a bit more of coverage under different setup than CI).

Reply:
If the settings are in testcase.yaml file, then you cannot run the test directly from command line.

Reply:
No, you can run *this test* from command line. Just try, and see.

Reply:
Been there, done that. Sure it works if you setup slip etc. Running the test standalone requires CONFIG_NET_TEST=y and CONFIG_NET_LOOPBACK=y so it makes sense to place them to prj.conf so that they are always available.

Reply:
> Sure it works if you setup slip etc.

Yes. And if usually run slip setup anyway, then it should be pretty seamless. I thus would suggest to avoid making changes unless really needed - again, having it both ways allows for better coverage. That's just a suggestion though. (But other things in this PR are truly -1's.)

Reply:
> And if usually run slip setup anyway, then it should be pretty seamless. I thus would suggest to avoid making changes unless really needed - again, having it both ways allows for better coverage.

That is just your usecase, for me I usually do not use slip any longer. So NAK to this comment.